### ISSUE TITLE
Investigate whether the tool can use power shell unzip instead of package:archive

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#45278]
package:archive is used to provide zip/unzip only on Windows host machines. Since Windows 8 and power shell 3, the native zip/unzip functionality has been exposed and could be used instead of the dart implementation.

For reference https://stackoverflow.com/questions/17546016/how-can-you-zip-or-unzip-from-the-script-using-only-windows-built-in-capabiliti/26843122#26843122

**Comment by christopherfujino** [comment]
What are the advantages of using native unzip? (I'm assuming speed?) I've seen a lot of issues where operations failed because external tools we assumed would be there weren't.

**Comment by jonahwilliams** [comment]
It would allow us to a) Remove our dependency on package-archive which uses a Dart implementation of unzip and b) avoid round-tripping multiple MB of data through the Dart heap when doing so

**Comment by jonahwilliams** [comment]
TLDR: no
